helpers.ts

```diff
@@ -337,14 +337,41 @@ export function parseLongitude(lon: string, hemi: string): number {
     return parseDmCoordinate(lon) * hemisphere;
 }
 
+function getYearFromString(yearString: string, rmcCompatible: boolean): number {
+    if (yearString.length === 4) {
+        return Number(yearString);
+    } else if (yearString.length === 2) {
+        if (rmcCompatible) {
+            // GPRMC date doesn't specify century. GPS came out in 1973 so if the year
+            // is less than 73, assume it's 20xx, otherwise assume it is 19xx.
```

Comment:
This makes sense for historic parsing, but thinking about it now, is that useful?  This is fine, but I'd also be OK with you just using the simple case of always adding "20" (and updating the test accordingly).

Reply:
It all depends on the use-case. If people want to parse logs form the 90's this is essential.  I would leave it on for the upcoming 40-50 years.

(I can merge, but I'll wait for your approval on this thread)

Reply:
OK, let's leave it.